src/index.css

```diff
@@ -8,5 +8,51 @@
   font-family: 'Work Sans', sans-serif;
 }
 
+/* Global Styles */ 
 
+/* Ensures a parent element with this class name stretches to the height of floated children */
+.clearfix:after {
```

Comment:
This is a SUPER helpful class. Block elements automatically stretch to fit the height of any non-floated static or relative positioned elements. But, if you give an element a float, then the parent block element doesn't expand to the children's height without some css. One way is to give the parent an `overflow: hidden` or `overflow: auto`. But, relying on overflow's can be tricky, if you ever want to purposefully have a visible overflow. So, this classname adds a pseudo-element to the parent that forces it to stretch to the height of even floated child elements. 

Note: parent elements will never stretch to absolute or fixed position children.